plugins/janus_streaming.c

```diff
@@ -6519,6 +6521,23 @@ static int janus_streaming_rtsp_connect_to_server(janus_streaming_mountpoint *mp
 	}
 	long code = 0;
 	res = curl_easy_getinfo(curl, CURLINFO_RESPONSE_CODE, &code);
+#if CURL_AT_LEAST_VERSION(7, 62, 0)
+	if(code == 404) { //Possibly a quirk in the RTSP server, where the DESCRIBE request expects a path only.
+		CURLU *curl_u;
+		char *path;
+		curl_u = curl_url();
```

Comment:
I'd collapse the definition as `CURLU *curl_u = curl_url();`, no need for pre-definitions. Besides, `path` should be initialized to `NULL`, since otherwise it may point to random garbage and the `&path` may be dangerous.